From 37b9ecd1089aa1189303200366f4fdf39abd958f Mon Sep 17 00:00:00 2001
From: JinHuiEric Huang <JinHuiEric.Huang@amd.com>
Date: Mon, 6 Mar 2017 14:31:16 -0500
Subject: [PATCH 0704/2094] Revert "drm/amd/powerplay: add voltage change
 support through pp_table"

This reverts commit 67819ca7018ea07c7184972ec90c41036da73c0e.

Change-Id: Ic7bc29df690fca322dd60d5b8110a1c35539acb7
---
 drivers/gpu/drm/amd/powerplay/amd_powerplay.c | 12 +-----------
 drivers/gpu/drm/amd/powerplay/inc/hwmgr.h     |  1 -
 2 files changed, 1 insertion(+), 12 deletions(-)

diff --git a/drivers/gpu/drm/amd/powerplay/amd_powerplay.c b/drivers/gpu/drm/amd/powerplay/amd_powerplay.c
index b5edabd..fde8fcd 100644
--- a/drivers/gpu/drm/amd/powerplay/amd_powerplay.c
+++ b/drivers/gpu/drm/amd/powerplay/amd_powerplay.c
@@ -750,17 +750,7 @@ static int pp_dpm_set_pp_table(void *handle, const char *buf, size_t size)
 
 	hwmgr->soft_pp_table = hwmgr->hardcode_pp_table;
 
-	ret = amd_powerplay_reset(handle);
-	if (ret)
-		return ret;
-
-	if (!hwmgr->hwmgr_func->avfs_control) {
-		ret = hwmgr->hwmgr_func->avfs_control(hwmgr, false);
-		if (ret)
-			return ret;
-	}
-
-	return 0;
+	return amd_powerplay_reset(handle);
 }
 
 static int pp_dpm_force_clock_level(void *handle,
diff --git a/drivers/gpu/drm/amd/powerplay/inc/hwmgr.h b/drivers/gpu/drm/amd/powerplay/inc/hwmgr.h
index d5aa6cd..8cf5aed 100644
--- a/drivers/gpu/drm/amd/powerplay/inc/hwmgr.h
+++ b/drivers/gpu/drm/amd/powerplay/inc/hwmgr.h
@@ -360,7 +360,6 @@ struct pp_hwmgr_func {
 	int (*release_firmware)(struct pp_hwmgr *hwmgr);
 	int (*set_power_profile_state)(struct pp_hwmgr *hwmgr,
 			struct amd_pp_profile *request);
-	int (*avfs_control)(struct pp_hwmgr *hwmgr, bool enable);
 };
 
 struct pp_table_func {
-- 
2.7.4

